.github/workflows/build-commit-run-tpch.yaml

```diff
@@ -0,0 +1,51 @@
+name: Build commit and run tpch benchmarks
+
+on:
+  workflow_dispatch:
+    inputs:
+      skip_questions:
+        description: The TPC-H questions to skip
+        required: false
+        default: ""
+      scale_factor:
+        description: Which scale factor to use
+        required: false
+        type: choice
+        options:
+        - '2'
+        - '10'
+        - '100'
+        - '1000'
+        default: '2'
+      partition_size:
+        description: Which partition size to use
+        required: false
+        options:
+        - '2'
+        - '32'
+        - '32'
```

Comment:
I think  we should combine SF with partition size, since not all permutations have datasets?

I.e. `SF10_part32`, `SF100_part512`...

Or even simpler, just let people select the URL

Reply:
Ya, I was thinking of this as well.

Options could be:
- `2-2`
- `10-32`
- etc.

Reply:
Will update.